src-docs/src/views/home/home_view.js

```diff
@@ -52,6 +44,16 @@ export const HomeView = () => (
               <EuiIcon type="logoGithub" aria-hidden="true" />
             </EuiLink>
           </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <p>
+              Version:{' '}
+              <Link
+                aria-label={`Version ${pkg.version}, View changelog`}
+                to="/package/changelog">
+                <strong>v{pkg.version}</strong>
```

Comment:
Prefacing this now with `Version:` makes the `v` here duplicative. 
```suggestion
                <strong>{pkg.version}</strong>
```